src/data/fp/basic.lean

```diff
@@ -1,8 +1,3 @@
-/-
```

Comment:
I've filed a github bug about `end fp` appearing here, their line number indices are screwed up.

Reply:
This seems to have gone away after the last push...